build.py

```diff
@@ -109,16 +109,20 @@ def create_package_fs(build_root):
         create_dir(os.path.join(build_root, d))
         os.chmod(os.path.join(build_root, d), 0755)
 
-def package_scripts(build_root):
+def package_scripts(build_root, static=False):
```

Comment:
So if I understand correctly by passing static here we don't copy all the service scripts etc just the config. Then for builds marked static and all windows build we set static=True since these extra scripts don't apply.

The name `static` for this method is confusing maybe we should name this parameter `config_only`?

Then this code below dosen't read as all windows builds are static:

```python
if platform == 'windows' or static:
    package_scripts(build_root, static=True)
```

It is a simple fix but it was quite confusing at first. 

Also why do static build not need the extra service scripts etc? Is it because they are intended for use in apline linux? If so a comment explaining that would be helpful.